MIGRATION.MD

```diff
@@ -0,0 +1,8 @@
+# Migration guides
+
+## 3.1.x to 3.2.0
+- `Radar.setAdIdEnabled()` has been removed.
+- Custom events have been renamed to conversions.
+    - `Radar.sendEvent(customType, location, metadata)` is now `Radar.logConversion(name, revenue, metadata)`.
+    - The method response does not include `location` and `user` props anymore.
```

Comment:
Can you add code snippets below of the migration, for example, how this is done in the native SDKs: https://github.com/radarlabs/radar-sdk-android/blob/master/MIGRATION.md#36x-to-370

Reply:
Yes. I will

Reply:
@tjulien added code snippets